### PR TITLE
Resolve #254

### LINKED ISSUE #254: Add placeholder for empty screens
**Describe the feature you'd like**

Whenever a page does not have any content to display, for example, an empty RecyclerView, then instead of a blank screen, a 'Nothing to display' or similar message/image should be displayed to the user.

**Would you like to work on the issue?**

No.

### PR DESCRIPTION
**Pull request by ShcherbaDev**
## What this PR does
This is a resolve for issue #254. Added placeholders for empty screens and modified some already existing placeholders.

Full changes list:

- In fragment "MyDashboardFragment" now can be show a TextView with id "textViewNoCourses" if user haven't joined a program or have happen an error in API.
- Created `@string/tv_no_courses` which appears when was happen a failure in loading courses data or courses list is empty.
- Content of a TextView with id "textViewNoCourses" in `fragment_explore_course_list.xml` was changed from `@string/hello_blank_fragment` to `@string/tv_no_courses`.
- Deleted a cookie text for TextView with id "textViewNoCourses" in fragments `CourseListFragment` and `ProfileCourseListFragment`.
- Modified `@string/not_entolled` from "You are not enrolled in any course" to "Not enrolled in any course".

## Screenshots
Before:
Before 01
Before 02
Before 03
Before 03 (1)
Before 04

After:
After 01
After 02-03
After 03 (1)
After 04

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -58,7 +59,7 @@
     <string name="references_added">References Added</string>
     <string name="outreach_programs_and_events_dashboard">Outreach Programs and Events Dashboard</string>
     <string name="menu_settings">Settings</string>
-    <string name="not_enrolled">You are not enrolled in any course</string>
+    <string name="not_enrolled">Not enrolled in any course</string>
```
Comment (ShridharGoel) [inline app/src/main/res/values/strings.xml]: Unnecessary change.

Comment (ShcherbaDev) [reply app/src/main/res/values/strings.xml]: Got it. I returned the previous content of this string.
